_data/terms-of-service.yml

```diff
@@ -73,35 +73,35 @@
 
     - title: 6. Intellectual Property Rights
       legalese: |
-        <p id="customer-license"><strong>6.1. Customer License.</strong> Subject to the terms of service of the applicable specific Services you have subscribed to, and provided that you pay Gruntwork all amounts owed pursuant to these Terms and such other terms of service and that you are not in breach of any provision of these Terms or such other terms of service, Gruntwork hereby grants to you a royalty-free, worldwide, non-exclusive, non-transferable (other than as specifically set forth in <a href="#15-assignment">section 15</a> below), non-sublicensable license for your Authorized Users to use, install, test, execute, perform, and copy the Services exclusively for your business use, and to create derivative works or otherwise modify the Services (including source code of the Services) purely for your own purposes (the “License”).</p>
+        <p id="customer-license"><strong>6.1. Customer License.</strong> Subject to the terms of service of the applicable specific Services you have subscribed to, and provided that you pay Gruntwork all amounts owed pursuant to these Terms and such other terms of service and that you are not in breach of any provision of these Terms or such other terms of service, Gruntwork hereby grants to you a royalty-free, worldwide, non-exclusive, non-transferable (other than as specifically set forth in <a href="#15-assignment">section 15</a> below), non-sublicensable license for your Authorized Users to use, install, test, execute, perform, and copy the Services exclusively for your business use, and to create derivative works or otherwise modify the Services (including source code of the Services) purely for your own purposes (the “License”). <i>For clarity, the License is granted on a per-user basis; and you must purchase and maintain a valid license, subscription or similar access rights for any and all users who access the Gruntwork code, whether via a Gruntwork code repository, private forked copy of a code repository, or third-party software system, such as in the case of a continuous integration system that may call upon Gruntwork code or any derivatives thereof.</i></p>
```

Comment:
```suggestion
        <p id="customer-license"><strong>6.1. Customer License.</strong> Subject to the terms of service of the applicable specific Services you have subscribed to, and provided that you pay Gruntwork all amounts owed pursuant to these Terms and such other terms of service and that you are not in breach of any provision of these Terms or such other terms of service, Gruntwork hereby grants to you a royalty-free, worldwide, non-exclusive, non-transferable (other than as specifically set forth in <a href="#15-assignment">section 15</a> below), non-sublicensable license for your Authorized Users to use, install, test, execute, perform, and copy the Services exclusively for your business use, and to create derivative works or otherwise modify the Services (including source code of the Services) purely for your own purposes (the “License”). For clarity, the License is granted on a per Authorized User basis, and you must purchase and maintain a valid license for each Authorized User who accesses the Gruntwork Services, no matter how they access the Gruntwork Services, including, but not limited to, access via Gruntwork's code repositories and access via a private copy of a code repository.</p>
```

Reply:
LGTM! Thanks, I think this is ready to deploy.

Reply:
Looks good. Will commit and deploy.